c/pthread-C-DAC/ALL.prp

```diff
@@ -0,0 +1,2 @@
+CHECK( init(main()), LTL(G ! call(__VERIFIER_error())) )
```

Comment:
Please remove this file, the meta data structure in the repository has changed in the meantime.